about.html

```diff
@@ -88,21 +88,10 @@ <h2>News about Beijing 2022 Winter Olympics Games</h2>
                 </li>
             </ul>
         </section>
-        <section id="Sponsors" class="sponsors-section">
-            <h2 class="sponsors-title">Sponsors</h2>
-            <ul class="sponsors-list header-2 header-3">
-                <li><img src="images/visa-master.png" alt="visa-master"></li>
-                <li><img src="images/google-logo.png" alt="google-logo"></li>
-                <li><img src="images/Toyota-Logo.png" alt="Toyota-Logo"></li>
-                <li><img src="images/mozilla.png" alt="mozilla"></li>
-                <li><img src="images/airbnb.png" alt="airbnb"></li>
-                <li><img src="images/naver.png" alt="naver"></li>
-            </ul>
-      </section>
     </main>
     <footer id="footer" class="footer-container header-2">
```

Comment:
- [ ] [OPTIONAL] You have done a great job however you need to align the contents of the desktop footer for both the Home and the About page to the center just the way you did for the sponsors.
### Your Implementation
![Screenshot from 2022-04-13 18-44-44](https://user-images.githubusercontent.com/58692590/163240915-74167519-903b-4b73-ba8e-79b05fcf086a.png)